src/components/Link/README.md

```diff
@@ -0,0 +1,192 @@
+`}>
+    <UIKit.Link view="primary" href="/">Link</UIKit.Link>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Link view="primary" href="/">
+  Link
+</Link>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Secondary
+
+Like primary links, they are used when it is natively clear to the user that an element can be interacted with, and the need to navigate through them is secondary and affects a small number of scenarios. The main task is not to argue with the accents on the page. It is more often used in bread crumbs or when displaying secondary attributes.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Link view="secondary" href="/">Link</Link>
+`}>
+    <UIKit.Link view="secondary" href="/">Link</UIKit.Link>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Link view="secondary" href="/">
+  Link
+</Link>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Visitable
+
+Used to show that the link has already been visited.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Link href="https://gravity-ui.com/" visitable>Link</Link>
+`}>
+    <UIKit.Link href="https://gravity-ui.com/" visitable>Link</UIKit.Link>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Link href="https://gravity-ui.com/" visitable>
+  Link
+</Link>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Href
+
+`href` property is optional. If it is absent, then Link wil act like Button.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Link href="/">Link with href</Link>
+<Link>Link without href</Link>
+`}>
+    <UIKit.Link href="/">Link with href</UIKit.Link>
+    <UIKit.Link>Link without href</UIKit.Link>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Link href="/">Link with href</Link>
+<Link>Link without href</Link>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Usage
+
+Сan be used as an independent text element, or as part of the text
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Text>
+    <Link>what roles are active in the service</Link>
+</Text>
+<Text>
+    Currently, this role can only be assigned to a <Link>folder</Link> or <Link>cloud</Link>
+</Text>
+`}>
+    <UIKit.Text>
+        <UIKit.Link>what roles are active in the service</UIKit.Link>
+    </UIKit.Text>
+    <UIKit.Text>
+        Currently, this role can only be assigned to a <UIKit.Link>folder</UIKit.Link> or <UIKit.Link>cloud</UIKit.Link>
+    </UIKit.Text>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Text>
+    <Link>what roles are active in the service</Link>
+</Text>
+<Text>
+    Currently, this role can only be assigned to a <Link>folder</Link> or <Link>cloud</Link>
+</Text>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
+
+| Name       | Description                             | Type                                                                         | Default    |
+| :--------- | :-------------------------------------- | :--------------------------------------------------------------------------- | :--------- |
```

Comment:
Let's align type and default column in center

Reply:
It's not about formatting the code, it's about formatting content. How it can be linted?

Reply:
Thought to meant markup

Reply:
Done

Reply:
@LakeVostok  Now it's description column centered, but type and default only should be

Reply:
Like this?

Reply:
@amje

Reply:
yep